### DISCUSSION TITLE
Can this library be used with React Native?

### DISCUSSION THREAD

**Discussion by astanb** [FirebaseExtended/reactfire#465] [Q&A] [answered]
I can't find any definitive information on whether this is possible. I only found an issue redirecting someone to react-native-firebase but with no more context than that. Please can someone confirm whether it is usable or not? Thanks

**Comment by jhuleatt** [comment, marked as answer]
Hi @alexstanbury, building an app with React Native requires a library that can call through to the Firebase Android and iOS SDKs. ReactFire doesn't have that capability, so [react-native-firebase](https://rnfirebase.io/) is a better fit for React Native apps.

**Reply by astanb** [reply]
Thanks for the quick reply.

I have just found out that the JavaScript web SDK can be used with react native, and I've just got the latest version 8 of that working with reactfire 3.0.0 (seems to work after a brief preliminary test of the useAuth hook anyway!).

The web SDK has less features than the native sdks but they aren't of use to me anyway. 

I ran into an issue with the SDK 9 version of the js web SDK and reactfire 4 but I think SDK 8 will be fine for a native app as the tree shaking stuff is less important.
